packages/terra-clinical-application/README.md

```diff
@@ -0,0 +1,32 @@
+# Terra Clinical Application
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-clinical-application.svg)](https://www.npmjs.org/package/terra-clinical-application)
+[![Build Status](https://travis-ci.org/cerner/terra-ui.svg?branch=master)](https://travis-ci.org/cerner/terra-ui)
+
+The Application is a wrapper component around `terra-base` that provides clinical application-specific properties.
```

Comment:
I updated it in the doc/README: https://github.com/cerner/terra-clinical/pull/42/commits/134f810feaa3bb40bc1c502347a37f46f7b318c3